lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -688,6 +720,23 @@ def add_3ds2_authenticated_data(post, options)
         }
       end
 
+      def add_fund_source(post, options)
+        return unless fund_source = options[:fund_source]
```

Comment:
The also have a `card` object but wasn't sure that we want customers to send us card info like this